test/loopback.test.js

```diff
@@ -94,6 +95,7 @@ describe('loopback', function() {
         'template',
         'token',
         'urlNotFound',
+        'urlencoded',
```

Comment:
This failing test has been already fixed by https://github.com/strongloop/loopback/pull/3637 in a way that should be more resilient to future changes. I'll revert this file as part of final cleanup of the patch.